packages-native/animation/src/Animation.tsx

```diff
@@ -40,7 +40,7 @@ export class Animation extends Component<Props> {
 
     private validateProps(props: Props): void {
         const { afterAnimationAction, count } = props;
-        if (afterAnimationAction && count === 0) {
+        if (afterAnimationAction && count.eq(0)) {
```

Comment:
Throws an error: 
![image](https://user-images.githubusercontent.com/7356299/78683276-611c8f80-78ef-11ea-9ca3-195a53a6053b.png)

Reply:
Should be fixed now